### DISCUSSION TITLE
Import utility

### DISCUSSION THREAD

**Discussion by jrfcs** [nguyenquyhy/Flight-Recorder#103] [Ideas]
I'm looking for a way to convert CSV to fltrec format.    Any write one?   I exported just fine but it would be nice to modify the CSV and import it back for replay.   Thoughts, anyone?   If not here, does anyone know of an addin that can do that?

**Comment by jrfcs** [comment]
I would be willing to sponsor or pay for such a feature

**Comment by nguyenquyhy** [comment]
I thought about this before as it is a nice way to allow our users to modify the data at will. The challenge here is that the CSV file does not have all the information in .fltrec file (e.g. file version, aircraft title), so a simple import won't be the same as a recording.

Do you have any specific modification you have in mind? I am adding some basic editing feature directly in the tool such as trimming, so that might be a viable solution for now.

I have created an issue to track this #104. I'll try to think more about this around end Dec or Jan as I am too busy around this holiday period and cannot manage much time to add features for the tool right now.
